packages/dart/lib/src/objects/parse_object.dart

```diff
@@ -271,7 +290,7 @@ class ParseObject extends ParseBase implements ParseCloneable {
         final ParseResponse response = await batchRequest(
           requests,
           chunk,
-          client: _client,
+          client: client,
```

Comment:
```suggestion
          client,
```

Reply:
This suggestion is not correct, it creates a conflict.